### ISSUE TITLE
ipcam.settings

### ISSUE THREAD

**Issue by Dinuirar** [niqdev/ipcam-view#16]
I installed ipcam-view, but I'm getting an error when I'm trying to compile example program.

"Cannot resolve ipcam" because of:

```
import static com.github.niqdev.ipcam.settings.SettingsActivity.PREF_AUTH_PASSWORD;
import static com.github.niqdev.ipcam.settings.SettingsActivity.PREF_AUTH_USERNAME;
import static com.github.niqdev.ipcam.settings.SettingsActivity.PREF_IPCAM_URL;

```

Should I install some additional software, or did I install ipcam incorrectly?

**Comment by niqdev** [comment]
Hi!
no additional library is required. Sorry but when you say "install", you mean add the gradle dependency in the `build.gradle` of your app, right?
Can you provide a complete stacktrace please, otherwise I don't understand your error.
Thanks

**Comment by Dinuirar** [comment]
I downloaded ipcam-view-master.zip, unpacked in my home folder and in script I changed path to ndk-build. I runned the compileJni.sh script. I added the gradle dependency: 
in build.gradle (module app):
```
dependencies {
    compile 'com.github.niqdev:mjpeg-view:0.3.6'
}
```
and in build.gradle (Project: test):
```
allprojects {
    repositories {
        jcenter()
    }
}
```
Do I need butterknife from https://github.com/JakeWharton/butterknife to run example app?

**Comment by niqdev** [comment]
So I did all the steps from scratch in order to help you:

I cloned the repo from command line with `git clone git@github.com:niqdev/ipcam-view.git`, if you don't have an SSH key use `git clone https://github.com/niqdev/ipcam-view.git`

then in the project root directory I added a file `local.properties` with `sdk.dir=/YOUR/PATH/Android/Sdk` 

and then compiled the project with `./gradlew build` (ignore the lint errors).

If you use Android Studio instead simply import the project and run the app in the emulator, it should works straight away.

You don't need to do anything else in order to run the sample app that is in the play store!
